dev/buildtool/build_commands.py

```diff
@@ -234,272 +306,127 @@ def __is_newer_repo(self, test_path, ref_path, log):
-      force_new: [bool] If true, clear any existing repository
-    """
-    git_dir = self.source_code_manager.get_local_repository_path(name)
-    original_repo_path = os.path.abspath(git_dir)
+    bintray_key = os.environ['BINTRAY_KEY']
+    bintray_user = os.environ['BINTRAY_USER']
+    if not bintray_key:
+      raise ValueError('Expected BINTRAY_KEY set.')
+    if not bintray_user:
+      raise ValueError('Expected BINTRAY_USER set.')
 
     options = self.options
-    nebula_repo_path = os.path.join(options.scratch_dir, 'build', name)
-
-    if os.path.exists(nebula_repo_path):
-      if not force_new:
-        logged_is_newer = self.__logged_is_newer.get(nebula_repo_path, False)
-        if self.__is_newer_repo(nebula_repo_path, original_repo_path,
-                                not logged_is_newer):
-          if not logged_is_newer:
-            logging.info('Reusing existing %s', nebula_repo_path)
-            self.__logged_is_newer[nebula_repo_path] = True
-          return nebula_repo_path
-
-      self.__logged_is_newer[nebula_repo_path] = False
-      logging.info('Removing old nebula repo %s', nebula_repo_path)
-      shutil.rmtree(nebula_repo_path, ignore_errors=True)
-
-    logging.debug('Determining version and build tag of %s',
-                  original_repo_path)
-    summary = self.git.collect_repository_summary(original_repo_path)
-    build_tag = '{version}-{build}'.format(
-        version=summary.version, build=options.build_number)
-
-    logging.info('Cloning %s to nebula repo %s', name, nebula_repo_path)
-    self.git.clone_repository_to_path(original_repo_path, nebula_repo_path)
-
-    # Nebula complains if the git tag version doesn't match the branch's
-    # version, i.e. in patch releases. As a workaround, we checkout the
-    # HEAD commit of the current branch in a 'detached HEAD' state so
-    # Nebula doesn't complain about our branch version and git tag
-    # version not matching.
-    self.git.reinit_local_repository_with_tag(
-        nebula_repo_path, build_tag,
-        'Snapshot from {path} to satisfy nebula.\n\n'
-        'commit: {commit}\ntag: {tag}\nversion: {version}'
-        .format(path=original_repo_path,
-                commit=summary.commit_id,
-                tag=summary.tag,
-                version=summary.version))
-    self.__nebula_repository_version[name] = summary.version
-    summary_path = os.path.join(
-        options.scratch_dir, name, '{name}-summary.yml'.format(name=name))
-    write_to_path(summary.to_yaml(with_commit_messages=False), summary_path)
-    return nebula_repo_path
+    jar_repo = options.build_jar_repository
+    if not jar_repo or len(jar_repo.split('/')) != 1:
+      raise ValueError('Expected --build_jar_repository in the form <repo>'
+                       ' using the implied owner from'
+                       ' --build_bintray_repository')
+
+    parts = (options.build_bintray_repository or '').split('/')
```

Comment:
It always seemed weird that we were explicitly passing a single jar repo string, but the org and deb package repo as `{org}/{deb_repo}`.

Reply:
I changed the config options. have a look.